app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1504,6 +1515,10 @@ DeviceDelegateOpenXR::DrawHandMesh(const uint32_t aControllerIndex, const vrb::C
   m.handMeshRenderer->Draw(aControllerIndex, aCamera);
 }
 
+void DeviceDelegateOpenXR::SetHitDistance(const float distance) {
+  m.furthestHitDistance = std::max(distance, m.furthestHitDistance);
```

Comment:
Nice, didn't know about that.

Reply:
Is this a thing with floats? Not sure what you really mean in this case

Reply:
It could be that with floats doesn't apply, but still it doesn't harm and it's more coherent with the cases where it applies (eg, objects with custom comparison functions)